### PR TITLE
[ACM 14735] Release 2.13 config changes for ACM Obs

### PR DESCRIPTION
**Pull request by coleenquadros**
https://issues.redhat.com/browse/ACM-14735

### CODE REVIEW

**Comment by coleenquadros** [comment]
backplane chnages are not included yet.

**Comment by coleenquadros** [comment]
@subbarao-meduri

**Review by moadz — APPROVED** [review]

**Comment by subbarao-meduri** [comment]
/pj-rehearse pull-ci-stolostron-multicluster-observability-operator-release-2.13-test-e2e

File: ci-operator/config/stolostron/multicluster-observability-operator/stolostron-multicluster-observability-operator-release-2.13.yaml [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+  skip_if_only_changed: ^docs/|hack/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO)$
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: multicluster-observability-operator
+    env:
+      ACM_RELEASE_VERSION: release-2.13
+      CLUSTERPOOL_GROUP_NAME: Core-Services
+      CLUSTERPOOL_HOST_NAMESPACE: acm-observability-usa
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: ocm-obs-usa-clusterpool
+      CLUSTERPOOL_HUB_FILTER: sno-4xlarge-416
```
Comment (subbarao-meduri) [inline ci-operator/config/stolostron/multicluster-observability-operator/stolostron-multicluster-observability-operator-release-2.13.yaml:117]: We should update this to sno-4xlarge-417 at some point.

**Pull request merged**